packages/atomic/src/components/atomic-search-interface/analytics-config.ts

```diff
@@ -4,40 +4,99 @@ import {
 } from '@coveo/headless';
 import {AnalyticsClientSendEventHook} from '@coveo/headless/node_modules/coveo.analytics';
 import {getAtomicEnvironment} from '../../global/environment';
+import {ObservableMap} from '@stencil/store';
+import {AtomicStore, getAllFacets} from '../../utils/store';
 
 type AnalyticsPayload = Parameters<AnalyticsClientSendEventHook>[1];
 
 export function getAnalyticsConfig(
   searchEngineConfig: SearchEngineConfiguration,
-  enabled: boolean
+  enabled: boolean,
+  store: ObservableMap<AtomicStore>
 ): AnalyticsConfiguration {
+  const analyticsClientMiddleware = (
+    event: string,
+    payload: AnalyticsPayload
+  ) => augmentAnalytics(event, payload, store, searchEngineConfig);
+
+  const outOfTheBoxConfig: AnalyticsConfiguration = {
```

Comment:
do you guys always use 'out of the box' in names, or would it be better to change to `default`?

Reply:
I would also prefer `default` or `fallback`.

Reply:
yeah sorry, I meant replacing `outOfTheBox` with `default`, so `defaultConfig`